src/autoscaler/metricscollector/db/metricscollector.db.changelog.yml

```diff
@@ -30,4 +30,46 @@ databaseChangeLog:
                   name: value
                   type: varchar
                   constraints:
-                    nullable: true
\ No newline at end of file
+                    nullable: false
+  - changeSet:
+      id: 2
+      author: byang
+      changes:
+        - createTable:
+            tableName: appinstancemetrics
```

Comment:
I will add index.

timestamp will give the right time metrics are reported, otherwise we can only use the time when we poll metrics from loggregator, which does not reflect the correct time of the metric.  And loggregator caches the metrics, so even the app instance is stopped, we still get a metric value when we poll before it expires in cache. 

'name' is reserved for other metric type.  Do we want to store different type of metrics in different table?  
Making it generic will be better for future extension with more metric types.